data/defaults.yaml

```diff
@@ -69,6 +69,14 @@ prometheus::config_dir: '/etc/prometheus'
 prometheus::config_mode: '0640'
 prometheus::config_show_diff: true
 prometheus::config_template: 'prometheus/prometheus.yaml.erb'
+prometheus::bind_exporter::download_extension: 'tar.gz'
```

Comment:
instead of adding the defaults here, could you add them directly to the bind_exporter class? that enables puppet-string to pick those values up and add them to the REFERENCE.md

Reply:
done